components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/constant/ConfigurationConstants.java

```diff
@@ -102,7 +112,16 @@
         ERROR_CODE_REPLACE_RESOURCE("CONFIGM_00034", "Error while replacing the resource : %s."),
         ERROR_CODE_REPLACE_ATTRIBUTE("CONFIGM_00035", "Error while replacing the attribute: %s."),
         ERROR_CODE_FEATURE_NOT_ENABLED("CONFIGM_00036", "Configuration management feature is not enabled."),
-        ;
+        ERROR_CODE_INSERT_FILE("CONFIGM_00037", "Error while adding the file: %s."),
+        ERROR_CODE_GET_FILE("CONFIGM_00038", "Error while retrieving the file: %s."),
+        ERROR_CODE_GET_FILES("CONFIGM_00039", "Error while retrieving files the resource: %s."),
+        ERROR_CODE_DELETE_FILES("CONFIGM_00040", "Error while deleting files for the resource id: %s."),
+        ERROR_CODE_DELETE_FILE("CONFIGM_00041", "Error while deleting the file with id: %s."),
+        ERROR_CODE_GET_FILES_BY_TYPE("CONFIGM_00042", "Error while retrieving files the resource type: %s."),
```

Comment:
This should be "Error while retrieving files **for** the resource type: %s."

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2690